hushline/settings/__init__.py

```diff
@@ -605,6 +616,38 @@ def update_smtp_settings() -> Response | str:
         flash("👍 SMTP settings updated successfully")
         return redirect(url_for(".index"))
 
+    @bp.route("/update-brand-primary-color", methods=["POST"])
+    @admin_authentication_required
+    @personal_server_required
+    def update_brand_primary_color() -> Response | str:
+        host_org = HostOrganization.fetch_or_default()
```

Comment:
I think this revision may swallow DB inconsistency errors during runtime. Perhaps the original with your added ``fetch`` instead of the previous ``default``?

https://github.com/scidsg/hushline/blob/06042b73b1b8367c9a117296d3addc114357e772/hushline/settings.py#L725-L727

Reply:
see responses in thread https://github.com/scidsg/hushline/pull/622#discussion_r1780066196